macos/Sources/App/macOS/AppDelegate.swift

```diff
@@ -486,15 +486,16 @@ class AppDelegate: NSObject,
 
         // Sync our auto-update settings. If SUEnableAutomaticChecks (in our Info.plist) is
         // explicitly false (NO), auto-updates are disabled. Otherwise, we use the behavior
-        // defined by our "auto-update" configuration.
-        if Bundle.main.infoDictionary?["SUEnableAutomaticChecks"] as? Bool != false {
-            updaterController.updater.automaticallyChecksForUpdates =
-                config.autoUpdate == .check || config.autoUpdate == .download
-            updaterController.updater.automaticallyDownloadsUpdates =
-                config.autoUpdate == .download
-        } else {
+        // defined by our "auto-update" configuration (if set) or fall back to Sparkle
+        // user-based defaults.
+        if Bundle.main.infoDictionary?["SUEnableAutomaticChecks"] as? Bool == false {
```

Comment:
Unlike some other languages, it's nice that Swift will only evaluate the lhs to `== false` if it's non-nil.